.env.example

```diff
@@ -1,3 +1,5 @@
 JSONRPC_URL=http://localhost:10002
 PRIVATE_KEYS=0x
 STAKING_CONTRACT_ADDRESS=0x
+MAX_VALIDATOR_COUNT=1
+MIN_VALIDATOR_COUNT=4
```

Comment:
These should be empty, so 
```bash
MAX_VALIDATOR_COUNT=
MIN_VALIDATOR_COUNT=
```

Reply:
Fixed 60f2db0ca9d9ddad06ae03b8b8f8cfa9e346dcd9